ios/FluentUI/Button/FluentButtonToggleStyle.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+/// `ToggleStyle` which configures the `Toggle` according to its state and design tokens.
+public struct FluentButtonToggleStyle: ToggleStyle {
+    public init() {}
+
+    @Environment(\.fluentTheme) private var fluentTheme: FluentTheme
+
+    public func makeBody(configuration: Configuration) -> some View {
+        SwiftUI.Button(action: {
+            configuration.isOn.toggle()
+        }, label: {
+            configuration.label
+        })
+        .buttonStyle(configuration.isOn ? buttonStylePressed : buttonStyle)
+    }
+
+    private var buttonStyle: FluentButtonStyle {
+        var style = FluentButtonStyle(style: .transparentNeutral)
+        style.overrideTokens(buttonTokens)
+        return style
+    }
+
+    private var buttonStylePressed: FluentButtonStyle {
```

Comment:
Kind of as a follow up to our flyweight convo, could these be static properties? Would that theoretically help with memory footprint optimization in some way?

Reply:
These can definitely be static, yep. I’ll do so now!

Reply:
Scratch that--downstream it depends on `pressedButtonTokens`, which relies on the local `fluentTheme`. And since none of these properties are stored (all calculated at call-time), there should be no memory impact either way. So I'll be leaving these as-is, unless there's an additional compelling reason not to!